examples/convert_little_endian_to_big.cpp

```diff
@@ -30,10 +30,10 @@ int main()
     std::cout << "in is: " << std::hex << in << std::endl;
     // If the host is little endian, the put function
     // should change the byte order (no change for big endian)
-    endian::convert_endian::put32(in, data);
+    endian::little_endian::put32(in, data);
```

Comment:
@petya2164 the code is copied from sak: https://github.com/steinwurf/sak/blob/master/test/src/test_convert_endian.cpp#L35 

But I will fix it